cmd/crowdsec/main.go

```diff
@@ -153,7 +153,7 @@ func LoadAcquisition(cConfig *csconfig.Config) error {
 	} else {
 		dataSources, err = acquisition.LoadAcquisitionFromFile(cConfig.Crowdsec)
 		if err != nil {
-			return errors.Wrap(err, "while loading acquisition configuration")
```

Comment:
any reason why you removed the Wrap here ? It feels like we loose information ? But maybe I missed sth ?

Reply:
Yes, there was already TMI :)